packages/itwinui-css/src/utils/component-action/link-box.scss

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../mixins';
+
+@mixin iui-link-box {
+  position: relative;
+  width: fit-content;
+  height: fit-content;
+  cursor: pointer;
+
+  &:where(:focus-within) {
```

Comment:
the focus styling looks good for the most simple case when tabbing, but it will also show for mouse users. the solution there is to use `:has(:focus-visible)` and then add `:focus-within` fallback for browsers that don't support `:has`.

for more advanced cases, it looks a bit weird. am i focusing the tile (main action) or the secondary action inside it?
![](https://user-images.githubusercontent.com/9084735/223844649-5476a823-3853-4806-9984-05097224f720.png)

this can be fixed using `&:has(.iui-link-overlay:focus-visible)` but the fallback approach for this will still have the same problem. maybe that's ok? firefox will support `:has` by Q2 2023.

Reply:
Added suggested css rules.
I think it's ok, users can see the difference in moving focus. Also screen reader does a good job showing which one is selected.